nova/virt/vmwareapi/vmops.py

```diff
@@ -1447,8 +1448,10 @@ def _get_vm_and_vmdk_attribs():
             if snapshot_ref:
                 self._delete_vm_snapshot(instance, vm_ref, snapshot_ref)
 
+
```

Comment:
additional empty line. did you run pep8 on your changes?

Reply:
Yep. I must have accidentally added that afterwards. Sorry!